src/extras.jl

```diff
@@ -135,19 +130,38 @@ function cut(x::AbstractArray{T, N}, breaks::AbstractVector;
         breaks = sort(breaks)
     end
 
-    if extend
-        min_x, max_x = extrema(x)
+    if extend === true
+        xnm = T >: Missing ? skipmissing(x) : x
+        length(breaks) >= 1 || throw(ArgumentError("at least one break must be provided"))
+        local min_x, max_x
+        try
+            min_x, max_x = extrema(xnm)
```

Comment:
in what "reasonable" case this would throw?

Reply:
When all values are missing (see error below). Checking `all(ismissing, x)` would be costly for the normal case so I figured `try... catch` is better.

Reply:
```
julia> extrema([missing])
(missing, missing)
```
it does not seem to error